apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/BubbleMessageBox.java

```diff
@@ -316,13 +317,13 @@ private void handleQuoteMessageBox() {
         }
     }
 
-    private Label createAndGetMessage() {
-        Label label = new Label();
+    private SelectableLabel createAndGetMessage() {
+        SelectableLabel label = new SelectableLabel(item.getMessage());
         label.maxWidthProperty().unbind();
         label.setWrapText(true);
         label.setPadding(new Insets(10));
         label.getStyleClass().addAll("text-fill-white", "medium-text", "font-default");
-        label.setText(item.getMessage());
+        label.setEditable(false);
```

Comment:
I think we should set that in the SelectableLabel by default to false.

Reply:
Beside that small part all looks good

Reply:
Addressed
PTAL
Thanks